### DISCUSSION TITLE
Can I get backup?

### DISCUSSION THREAD

**Discussion by Thenafi** [deta/cloud-docs#312] [General]
In case anything goes wrong can I have a backup of the base or the drive  to be on the safe side? 

If yes then this will make Deta.sh more reliable to me .

[Solved]

**Comment by jnsougata** [comment]
may be clone the current base frequently to another base....

**Reply by Thenafi** [reply]
Isn't it possible to have it saved locally?

**Reply by jnsougata** [reply]
yes! just fetch everything and save it inside a json file

**Reply by Thenafi** [reply]
🙂dumb me never thought of this idea 

😄Thank You

**Reply by axmc1** [reply]
If you resolved your issue, consider closing the question/marking it as solved.
While we talk backups, a kind [Open-Sourcerer](https://github.com/Harman-Sandhu/) made [DetaBaseUI](https://harmansandhu.tech/baseui), a website that has a bit nicer UI and allows you to download your entire Base as a .json file.
Mustafa, Deta's CEO is already working with Harman Sandhu to host DetaBaseUI on a Deta domain, so you know that it's trustworthy :)

**Reply by Thenafi** [reply]
Thank You 🤗
